src/main/java/com/impactupgrade/nucleus/service/segment/XeroAccountingPlatformService.java

```diff
@@ -251,6 +279,19 @@ public String createTransaction(AccountingTransaction accountingTransaction) thr
         }
     }
 
+    @Override
+    public List<String> updateOrCreateTransactions(List<AccountingTransaction> accountingTransactions) throws Exception {
+        Invoices invoices = new Invoices();
+        invoices.setInvoices(accountingTransactions.stream().map(this::toInvoice).toList());
+        try {
+            Invoices createdInvoices = xeroApi.updateOrCreateInvoices(getAccessToken(), xeroTenantId, invoices, SUMMARIZE_ERRORS, UNITDP);
+            return createdInvoices.getInvoices().stream().map(invoice -> invoice.getInvoiceID().toString()).toList();
+        } catch (Exception e) {
+            env.logJobError("Failed to upsert invoices! {}", getExceptionDetails(e));
+            throw e;
+        }
+    }
```

Comment:
@VSydor Don't we need a dr-hub PR to override this? dr-hub overrode the original `createTransaction` to turn TICKET transactions into a Bank Transaction (skipping the Invoice).